torch/_export/serde/serialize.py

```diff
@@ -1380,6 +1380,15 @@ def deserialize_node(self, serialized_node: Node, target: Callable) -> None:
             self.deserialize_outputs(serialized_node, fx_node)
             fx_node.meta.update(self.deserialize_metadata(serialized_node.metadata))
 
+            # assign nn_module_stack metadata to submodule/subgraph nodes for higher order ops
+            if serialized_node.target == "torch.ops.higher_order.cond":
```

Comment:
I don't think it's very generic to enumerate all the higher order ops here. If we're adding other HOOs in the future, we could fail silently here.

I think it's okay to loop over all the graph arguments and not special casing on these ops, maybe just use "torch.ops.higher_order" as common prefix to test.

Reply:
Thanks! That idea makes sense.

Reply:
@zhxchen17 could you help re-review? Just made the change.